### ISSUE TITLE
&#37;Path&#37; different with Explorer Integration

### ISSUE THREAD

**Issue by ConEmu** [ConEmu/old-issues#797]
_From [masta2ooo](https://code.google.com/u/masta2ooo/) on October 31, 2012 05:55:00_

OS version: Win7/x64 ConEmu version: 120727c x64
Far version: None _Bug description_ I have added a path to my SYSTEM Environment Variables and when i open cmd.exe (as restricted or admin) i see the additional path in &#37;path&#37; with "set path".
When i open "C:\Program Files\ ConEmu \ConEmu64.exe" directly, the same path is shown.

Now i want to use the Explorer Integration by "Settings -> Features -> Integration -> ConEmu Here - Register".
When i run "set path" in the ConEmu instance opened by the explorer shell, i see completely different environment variables!
The command line of it is ""C:\Program Files\ ConEmu \ConEmu64.exe" /cmd cmd -cur_console:n". If i open the ConEmu myself from cmd with this command line i see the path variables as before (correct).

Does the Explorer have different environment variables and pass them to the ConEmu on opening?

regards, Bernhard

_Original issue: http://code.google.com/p/conemu-maximus5/issues/detail?id=797_

**Comment by ConEmu** [comment]
_From [ConEmu.M...@gmail.com](https://code.google.com/u/116287257743594353797/) on October 31, 2012 06:13:01_

Don't know exactly what Explorer do with its %PATH%, but of course env.vars will be inherited from parent process.

Two questions
1) What is "completely different"?
2) Try "Cmd here" (not sure with exact name, but this item will be shown by Explorer on _Shift_+RClick). What %PATH% will be shown?

PS. Did you logOff/logOn? May be explorer had not refresh its environment...

**Comment by ConEmu** [comment]
_From [masta2ooo](https://code.google.com/u/masta2ooo/) on October 31, 2012 06:59:03_

wow, you are fast!
after i reported the issue i thought the same as you: "maybe i changed the path and didnt restarted yet."
i was just restarting to proof my foolishness and you already replied.

tl;dr;
yes, i did not restarted the explorer process / restarted the machine.

thanks for the help :)

**Comment by ConEmu** [comment]
_From [ConEmu.M...@gmail.com](https://code.google.com/u/116287257743594353797/) on October 31, 2012 07:18:45_

:D

**Status:** Invalid
